source/js/front/helpers/likeHelpers.js

```diff
@@ -0,0 +1,45 @@
+export function getLocalStorage() {
```

Comment:
Rename function to make purpose more clear.
```suggestion
export function getLikedPostsFromLocalStorage() {
```